integration_tests/src/main/python/delta_lake_merge_test.py

```diff
@@ -58,7 +58,10 @@ def setup_dest_tables(spark, data_path, dest_table_func, use_cdf, partition_colu
             sql_text = "CREATE TABLE delta.`{path}` ({ddl}) USING DELTA".format(path=path, ddl=ddl)
             if partition_columns:
                 sql_text += " PARTITIONED BY ({})".format(",".join(partition_columns))
-            sql_text += " TBLPROPERTIES (delta.enableChangeDataFeed = true)"
+            if enable_deletion_vectors:
+                sql_text += " TBLPROPERTIES (delta.enableChangeDataFeed = true, delta.enableDeletionVectors = true)"
```

Comment:
Similar to previous comment, is CDF required for deletion vectors?  If not, this needs work for the non-CDF case.